nuxt.config.js

```diff
@@ -197,21 +197,21 @@ export default {
         file: 'pt-BR.js',
         name: 'Português',
         domain: 'https://pt.nuxtjs.org'
-      }
+      },
       // {
       // code: 'tr',
       // iso: 'tr-TR',
       // file: 'tr-TR.js',
       // name: 'Türkçe',
       // domain: 'https://tr.nuxtjs.org'
       // }
-      // {
-      //   code: 'es',
-      //   iso: 'es-ES',
-      //   file: 'es-ES.js',
-      //   name: 'Español',
-      //   domain: 'https://es.nuxtjs.org'
-      // }
+      {
```

Comment:
we need to leave this commented out for now until the domain is ready otherwise it will appear in the footer select with a broken link.

Reply:
i sorted it and added a comment for future contributors

Reply:
Great, thank you!